client/app/vps/vps.routing.js

```diff
@@ -16,11 +19,17 @@ export default /* @ngInject */($stateProvider) => {
       url: '/{serviceName}',
       redirectTo: 'iaas.vps.detail.dashboard',
       resolve: {
-        stateVps: /* @ngInject */ ($transition$, $q, OvhApiVps) => OvhApiVps.v6().get({
-          serviceName: _.get($transition$.params(), 'serviceName'),
+        capabilities: /* @ngInject */ (
+          serviceName,
+          OvhApiVpsCapabilities,
+        ) => OvhApiVpsCapabilities.Aapi().query({ serviceName }).$promise
+          .then(capabilities => capabilities.map(capability => _.kebabCase(capability))),
+        serviceName: /* @ngInject */ $transition$ => $transition$.params().serviceName,
+        stateVps: /* @ngInject */ ($q, serviceName, OvhApiVps) => OvhApiVps.v6().get({
```

Comment:
this looks more like `vps` than `stateVps` ?

Reply:
It implies many refactoring of the vps section. Ok for changing the name, but not directly in this PR :)

Reply:
Here is the sub-task to treat this : MANAGER-3735